lib/src/iterables/map_split.dart

```diff
@@ -0,0 +1,93 @@
+part of quiver.iterables;
+
+typedef bool mapPred<T>(T item);
+typedef S mapFunc<T, S>(T item);
```

Comment:
Here and below:

Typedefs should use the same [naming convention](https://dart.dev/guides/language/effective-dart/style#do-name-types-using-uppercamelcase) as other types. Generally, [avoid using abbreviations](https://dart.dev/guides/language/effective-dart/design#avoid-abbreviations) and favour full words. We should also use Dart 2 style typedefs (I've got a separate patch that migrates existing code to that style):
```dart
typedef MapPredicate<T> = bool Function(T item);
typedef MapFunction<T, S> = S Function(T item);
```